npk/src/npk.rs

```diff
@@ -18,6 +18,7 @@ use crate::{
     dm_verity::{append_dm_verity_block, Error as VerityError, VerityHeader, BLOCK_SIZE},
     manifest::{Manifest, Mount, MountFlag, Version},
 };
+use const_format::concatcp;
```

Comment:
Do we need another dependency for just doing constant strings that never change? If I got it right it's just used for `FS_IMG_NAME`, `MANIFEST_NAME` and `SIGNATURE_NAME`